crates/bitwarden-vault/src/cipher/cipher.rs

```diff
@@ -584,6 +584,31 @@ impl CipherView {
         let res = creds.decrypt(ctx, ciphers_key)?;
         Ok(res)
     }
+
+    /// Decrypts an encrypted key value using the cipher's key
+    ///
+    /// This method is a temporary solution to allow typescript client access to decrypted key
+    /// values, particularly for FIDO2 credentials.
+    ///
+    /// # Arguments
+    /// * `enc_key` - The encrypted key value to decrypt
+    /// * `ctx` - The key store context to use for decryption
+    ///
+    /// # Returns
+    /// * `Ok(String)` - The decrypted key value
+    /// * `Err(CryptoError)` - An error occurred during decryption
+    pub fn decrypt_key(
+        &self,
+        enc_key: EncString,
```

Comment:
suggestion: could we make this a bit more specific/limited maybe? E.g. `decrypt_fido2_private_key` so it doesn't just take a generic enc string?

Reply:
It seems you changed the name but the function itself still takes an arbitrary `EncString` and decrypts it